Berksfile

```diff
@@ -3,6 +3,7 @@ source 'https://supermarket.chef.io'
 solver :ruby, :required
 
 cookbook 'apache2', '< 6.0.0'
```

Comment:
Please remove this.

Reply:
So actually it looks like we should keep this but update it to be ``< 8.2`` As 8.2 and above require Chef 15 which breaks ChefSpec on 14.